docs/participate/run-nodes/full-node.mdx

```diff
@@ -222,353 +157,23 @@ Otherwise, [check troubleshooting section](/participate/run-nodes/full-node#stat
-
-### Hardware requirements
-
-Compared to a [validator](/participate/run-nodes/full-node#hardware-requirements), a liteserver mode requires less resources. However, it is still recommended to use a powerful machine to run a liteserver.
-
-- at least 16 cores CPU
-- at least 128 GB RAM
-- at least 1TB GB NVMe SSD _OR_ Provisioned 32+k IOPS storage
-- 1 Gbit/s network connectivity
-- 16 TB/month traffic on peak load
-- public IP address (_fixed IP address_)
-
-#### Recommended Providers
-
-Feel free to use cloud providers listed in the [Recommended Providers](/participate/run-nodes/full-node#recommended-providers) section.
+### Uninstall mytonctrl
 
-Hetzner and OVH are forbidden to run a validator, but you can use them to run a liteserver:
-
-- __Hetzner__: EX101, AX102
-- __OVH__: RISE-4
-
-### Installation of liteserver
-
-1. Complete the previous steps to install [MyTonCtrl](/participate/run-nodes/full-node#step-by-step-instructions).
-
-2. Create a config file
-
-```bash
-MyTonCtrl> installer
-MyTonInstaller> clcf
-
-Local config file created: /usr/bin/ton/local.config.json
-```
-
-3. This file will help you to connect to your liteserver. Copy the config file located on the specified path to your home to save it.
-
-```bash
-cp /usr/bin/ton/local.config.json ~/config.json
-```
-
-4. Create an empty `config.json` file on your local machine.
-
-5. Copy the content from the console to your local machine `config.json` file.
-
-```bash
-cat ~/config.json
-```
-
-### Check the firewall settings
-
-First, verify the Liteserver port specified in your `config.json` file. This port changes with each new installation of `MyTonCtrl`. It is located in the `port` field:
-
-```json
-{
-  ...
-  "liteservers": [
-    {
-      "ip": 1605600994,
-      "port": LITESERVER_PORT
-      ...
-    }
-  ]
-}
-```
-
-If you are using a cloud provider, you need to open this port in the firewall settings. For example, if you are using AWS, you need to open the port in the [security group](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html).
-
-Below is an example of opening a port in the bare metal server firewall.
-
-#### Opening a port in the firewall
-
-We will use the `ufw` utility ([cheatsheet](https://www.cyberciti.biz/faq/ufw-allow-incoming-ssh-connections-from-a-specific-ip-address-subnet-on-ubuntu-debian/)). You can use the one you prefer.
-
-1. Install `ufw` if it is not installed:
-
-```bash
-sudo apt update
-sudo apt install ufw
-```
-
-2. Allow ssh connections:
-
-```bash
-sudo ufw allow ssh
-```
-
-3. Allow the port specified in the `config.json` file:
+Download script and run it:
 
 ```bash
-sudo ufw allow <port>
+wget https://raw.githubusercontent.com/ton-blockchain/mytonctrl/master/scripts/uninstall.sh
+sudo bash uninstall
```

Comment:
Why not `sudo bash /usr/src/mytonctrl/uninstall.sh`?

Reply:
'cause there is no information that it downloads script too :)